### PR TITLE
Video listing page

### PR DESCRIPTION
**Pull request by Maianki**
## Files to be added 
- [src/App.js](https://github.com/Maianki/ShuttleMotion-Video-Library/compare/video-listing-page?expand=1#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67)
- [src/components/Card/VideoCard/VideoCard.jsx](https://github.com/Maianki/ShuttleMotion-Video-Library/compare/video-listing-page?expand=1#diff-9445d61cedb1fcac13e59aff7c1549fec15b9f242551ade7b38996e9f1c46f54)
- [src/components/Navbar/Navbar.jsx](https://github.com/Maianki/ShuttleMotion-Video-Library/compare/video-listing-page?expand=1#diff-547f8328c15a9a29000b7ef4690ba4f1039d148d5c208e5a48d37a2acd21982e)
- [src/components/Navbar/navbar.module.css](https://github.com/Maianki/ShuttleMotion-Video-Library/compare/video-listing-page?expand=1#diff-547f8328c15a9a29000b7ef4690ba4f1039d148d5c208e5a48d37a2acd21982e)
- [src/components/Sidebar/Sidebar.jsx](https://github.com/Maianki/ShuttleMotion-Video-Library/compare/video-listing-page?expand=1#diff-547f8328c15a9a29000b7ef4690ba4f1039d148d5c208e5a48d37a2acd21982e)
- [src/components/Sidebar/sidebar.module.css](https://github.com/Maianki/ShuttleMotion-Video-Library/compare/video-listing-page?expand=1#diff-547f8328c15a9a29000b7ef4690ba4f1039d148d5c208e5a48d37a2acd21982e)
- [src/index.css](https://github.com/Maianki/ShuttleMotion-Video-Library/compare/video-listing-page?expand=1#diff-547f8328c15a9a29000b7ef4690ba4f1039d148d5c208e5a48d37a2acd21982e)
- [src/pages/VideoListing/VideoListing.jsx](https://github.com/Maianki/ShuttleMotion-Video-Library/compare/video-listing-page?expand=1#diff-547f8328c15a9a29000b7ef4690ba4f1039d148d5c208e5a48d37a2acd21982e)
- [src/pages/VideoListing/videolisting.module.css](https://github.com/Maianki/ShuttleMotion-Video-Library/compare/video-listing-page?expand=1#diff-547f8328c15a9a29000b7ef4690ba4f1039d148d5c208e5a48d37a2acd21982e)

### Screenshot of Listing Page
Screenshot (665)

### CODE REVIEW

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me! Your code looks very clean and easy to understand and the UI is also good.

File: src/components/Chips/Chips.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export function Chips({ categoryName }) {
+  return <div class='chip'>{categoryName}</div>;
```
Comment (Developers-codz) [inline src/components/Chips/Chips.jsx]: here it should be className

File: src/pages/VideoListing/VideoListing.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <Sidebar />
+      </section>
+      <section className={styles.main}>
+        <div className={styles.categoryChips}>
+          {[
+            "Matches",
+            "Badminton Tutorials",
+            "Badminton Trick Shots",
+            "Players Interview",
+          ].map((categoryName) => (
```
Comment (Developers-codz) [inline src/pages/VideoListing/VideoListing.jsx:36]: can make a variable for category names or in backend we have categories in DB you can fetch categories from there as well

**Review by Developers-codz — APPROVED** [review]
Just Added a suggestion, Everything  looks good, folder is well structured and readable code

File: src/components/Chips/Chips.jsx (hunk 1 above)
Comment (Maianki) [reply src/components/Chips/Chips.jsx]: Thanks for pointing that out. Will correct it.

File: src/pages/VideoListing/VideoListing.jsx (hunk 2 above)
Comment (Maianki) [reply src/pages/VideoListing/VideoListing.jsx:36]: Yes, I will do that once I integrate the backend. Using it till the time I fetch data from APIs.

**Pull request merged**